api/server.go

```diff
@@ -236,6 +237,58 @@ func (s *server) consensusUpdatesIndexHandler(jc jape.Context) {
 	jc.Encode(res)
 }
 
+func (s *server) miningGetBlockTemplateHandler(jc jape.Context) {
+	var req MiningGetBlockTemplateRequest
+	if jc.Decode(&req) != nil {
+		return
+	}
+
+	// TODO: add polling
```

Comment:
I'm not sure what you would be polling here. Probably fine but might also be a leftover TODO.

Reply:
It's the long polling mentioned in the description.